src/app/content/content.component.html

```diff
@@ -1,24 +1,9 @@
-<div class="content">
```

Comment:
I don't think this grid should be removed from the content component for now since its displaying multiple items(Or you can create an **Items-list component** instead)
The Item-component should represent only one item